CMakeLists.txt

```diff
@@ -264,6 +264,9 @@ option(USE_ZSTD "Use ZSTD" OFF)
 cmake_dependent_option(
   USE_MKLDNN "Use MKLDNN. Only available on x86, x86_64, and AArch64." "${CPU_INTEL}"
   "CPU_INTEL OR CPU_AARCH64" OFF)
+cmake_dependent_option(
+  USE_MKLDNN_ACL "Use Compute Library for the Arm architecture." OFF
+  "USE_MKLDNN AND CPU_AARCH64 AND USE_MKLDNN_ACL" OFF)
```

Comment:
Why usemkldnnacl definition depends on itself?

Reply:
Ah, just a mistake - that's gone.